packages/ui/src/machines/authenticator/signUp.ts

```diff
@@ -100,16 +100,38 @@ export function createSignUpMachine({ services }: SignUpMachineOptions) {
                   entry: [sendUpdate(), 'clearError'],
                   invoke: {
                     src: 'signUp',
+                    onDone: [
+                      {
+                        cond: 'shouldSkipConfirm',
+                        target: 'skipCheck',
+                        actions: ['setUser'],
+                      },
+                      {
+                        target: 'resolved',
+                        actions: ['setUser', 'setCredentials'],
+                      },
+                    ],
+                    onError: {
+                      target: 'idle',
+                      actions: 'setRemoteError',
+                    },
+                  },
+                },
+                skipCheck: {
```

Comment:
Minor naming nit: maybe this can be named `skipConfirm` or `skipConfirmation` to align with the name `confirmSignUp`

Reply:
👍